app/controllers.rb

```diff
@@ -4,7 +4,20 @@
     render :home, layout: true, locals: {
       'title' => 'Open Data Maker',
       'endpoints' => DataMagic.config.api_endpoint_names,
-      'examples' => DataMagic.config.examples
+      'examples' => DataMagic.config.examples,
+      'categories' => DataMagic.config.data['categories'].to_json
+    }
+  end
+end
+
+# Category page
+OpenDataMaker::App.controllers do
```

Comment:
Rather than starting a new `App.controllers` block, include the `get` in the same block started on line 2. (This will require moving the comments around a bit.)